src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationDetailViewController.swift

```diff
@@ -57,7 +57,7 @@ extension AppInformationDetailViewController {
 
 extension AppInformationDetailViewController: UITextViewDelegate {
 	func textView(_ textView: UITextView, shouldInteractWith url: URL, in characterRange: NSRange, interaction: UITextItemInteraction) -> Bool {
-		WebPageHelper.openSafari(withUrl: url, from: self)
+		UIApplication.shared.open(url, options: [:], completionHandler: nil)
```

Comment:
What was the reason to change this? Links were opened with the old code too.

Reply:
i assume the other implementation did not work for telephone numbers?

Reply:
This would need extra handling in this case. Hyperlinks should not jump out of the app, should they?

Reply:
That was also my concern. In other places (e.g., Onboarding Privacy), links are opened inside the app. With this, links are opened outside of the app. So the behaviour is changed.

Reply:
Will add extra handling in the existing method, then no crashes should occur (this comment kinda relates to all the other comments you guys have done 👍 )

Reply:
This was not addressed in the new code. Links are still jumping outside of the app. 

IMO this needs to be aligned with the PO, and either:

1. All links in the app are opened inside the app, or
2. All links in the app are opened outside the app.

Personally, in this point of time, I wouldn't align anything, I would just implement 1., as it's already done like that across the app.